wiz-outpost-lite/templates/_helpers.tpl

```diff
@@ -66,21 +66,45 @@ wiz.io/runner: {{ .runner | quote }}
 {{- $runner = $runner | kebabcase }}
 {{- $runnerID := get $values "runnerID" | default $runner }}
 
+{{/* Get module type based on runner name - using a variable since we can't define a template inside another template */}}
+{{- $moduleType := "" }}
+{{- if hasPrefix "remediation-" $runner -}}
+  {{- $moduleType = "remediation" }}
+{{- else if eq $runner "container-registry" -}}
+  {{- $moduleType = "container-registry" }}
+{{- else if hasPrefix "vcs-" $runner -}}
+  {{- $moduleType = "vcs" }}
+{{- else -}}
+  {{- fail (printf "Invalid runner name: %s. Runner name must start with 'remediation-', 'vcs-', or be 'container-registry'" $runner) -}}
+{{- end }}
+
 {{/* e.g. remediation-aws-rds-003 -> outpost-lite-runner-remediation
 container-registry -> outpost-lite-runner-container-registry
 */}}
 {{- $imageName := "" }}
-{{- if hasPrefix "remediation" $runner }}
+{{- if eq $moduleType "remediation" }}
   {{- $imageName = "outpost-lite-runner-remediation" }}
 {{- else }}
   {{- $imageName = dig "image" "name" (printf "outpost-lite-runner-%s" $runner) $values }}
 {{- end }}
 
 {{- $values = deepCopy $values }}
+
+{{/* Unify with module specific values */}}
+{{- $values = mergeOverwrite (deepCopy (get $.Values.modules $moduleType)) $values }}
 {{- $values = merge $values (dict "image" (dict "name" $imageName)) }}
 
-{{/* Unify with global .Values to be used inside a "with" statement */}}
-{{- $values = dict "runner" $runner "runnerID" $runnerID "Values" (merge $values (omit $.Values "runners")) -}}
+{{/* Unify with global values */}}
+{{- $values = mergeOverwrite (deepCopy (omit $.Values "runners")) $values }}
+
+{{/* Overwrite openshift values if runner set openshift to true */}}
+{{- if and $values.openshift }}
+{{- $values = merge $values $.Values.openshiftOverrides }}
```

Comment:
Don't we want mergeOverwrite here as well ?

Reply:
?

Reply:
Discussed - we'll keep as is, b/c we want the customer to be able to override any openshift overrides in the regular values